protocols/v2/const-sv2/README.md

```diff
@@ -2,10 +2,13 @@
 
 [![crates.io](https://img.shields.io/crates/v/const_sv2.svg)](https://crates.io/crates/const_sv2)
 [![docs.rs](https://docs.rs/const_sv2/badge.svg)](https://docs.rs/const_sv2)
+[![rustc+](https://img.shields.io/badge/rustc-1.75.0%2B-lightgrey.svg)](https://blog.rust-lang.org/2023/12/28/Rust-1.75.0.html)
+[![license](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)](https://github.com/stratum-mining/stratum/blob/main/LICENSE.md)
+[![docs.rs](https://codecov.io/gh/stratum-mining/stratum/branch/main/graph/badge.svg)](https://app.codecov.io/gh/stratum-mining/stratum/tree/main/protocols%2Fv2%2Fconst-sv2)
```

Comment:
Remove since this crate does not have unit tests and this is the global repo code cov:

```suggestion
```